ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node.hs

```diff
@@ -909,6 +926,20 @@ stdLowLevelRunNodeArgsIO RunNodeArgs{ rnProtocolInfo
       , llrnMaxCaughtUpAge = secondsToNominalDiffTime $ 20 * 60   -- 20 min
       , llrnMaxClockSkew =
           InFuture.defaultClockSkew
+      , llrnMaxRollbackAge = case gcCSJConfig rnGenesisConfig of
+          -- If CSJ is disabled, we are either syncing from trusted peers (Praos
+          -- mode), or the LoP already handles historical rollbacks.
```

Comment:
```suggestion
          -- mode), or the LoP already indirectly handles historical rollbacks.
```

Reply:
This prohibition would still benefit the syncing Genesis node, even without CSJ. And honest nodes should never trigger it.

So: do we really need to disable it based on this config?

Reply:
Added it to `GenesisConfig` separately and moved the comment there :+1: